pages/Threat_Modeling_Process.md

```diff
@@ -58,11 +58,11 @@ This information is documented in a resulting Threat Model document. It is also
 
 ### Step 2: Determine and Rank Threats
 
-Critical to the identification of threats is using a threat categorization methodology. A threat categorization such as [STRIDE][wiki-stride] can be used, or the Application Security Frame (ASF) that defines threat categories such as Auditing & Logging, Authentication, Authorization, Configuration Management, Data Protection in Storage and Transit, Data Validation, and Exception Management.
+Critical to the identification of threats is using a threat categorization methodology. A threat categorization such as [STRIDE][wiki-stride] can be used, or the Application Security Frame ([[ASF](https://pathlock.com/learn/what-are-application-security-frameworks/)](https://pathlock.com/learn/what-are-application-security-frameworks/)) that defines threat categories such as Auditing & Logging, Authentication, Authorization, Configuration Management, Data Protection in Storage and Transit, Data Validation, and Exception Management.
```

Comment:
All of the `wiki-stride` placeholders should probably point to: `https://en.wikipedia.org/wiki/STRIDE_%28security%29`

Reply:
I am looking into it, sorry for the inconvenience I caused.